### PR TITLE
Implemented test cases NMS_FT_24, 25, 26

### PR DESCRIPTION
**Pull request by kanchannms**
It includes test cases NMS_FT_24, NMS_FT_25 and NMS_FT_26

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by koshalt** [comment]
ok to test

**Comment by nmsyogesh** [comment]
raised JIRA bug for the this.

https://applab.atlassian.net/browse/NMS-187

**Comment by larubbio** [comment]
add to whitelist

On Mon, Jun 8, 2015 at 5:12 AM, yogesh kumar gupta <notifications@github.com

**Comment by frankhuster** [comment]
please merge

File: testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java [hunk 1]
```diff
@@ -782,5 +784,146 @@ public void verifyFT77()
...
+				null)); // content
+
+        String expectedJsonResponse = createFailureResponseJson("<callingNumber: Invalid>");
+
+        assertTrue(SimpleHttpClient.execHttpRequest(httpPost,
+				HttpStatus.SC_BAD_REQUEST, expectedJsonResponse,
+				ADMIN_USERNAME, ADMIN_PASSWORD));
+    }
+
+    @Ignore
```
Comment (koshalt) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: add JIRA ticket link?

File: testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java [hunk 2]
```diff
@@ -782,5 +784,146 @@ public void verifyFT77()
...
+		StringEntity params = new StringEntity(sb.toString());
+		httpPost.setEntity(params);
+		httpPost.addHeader("content-type", "application/json");
+		return httpPost;
+	}
+    
+	@Test
+    public void verifyFT24() throws IOException,
+			InterruptedException {
+		        /**
```
Comment (koshalt) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: also, fix the comment format while adding the jira link

**Comment by srivastavaAbhi** [comment]
Tracked with branch NMS.FT.KILKARICONTROLLERIT.
So closing this request.

**Pull request closed without merging**